code/src/Components/AlbumDiv.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import data from '../data.json';
+
+import { AlbumInfo } from './AlbumInfo';
+
+// The actual div containing all information such as image, artist name, album name and the correct URL-links
+export const AlbumDiv = () => {
+    return (
+        <>
+            {data.albums.items.map(album => { // Mapping through the album array to get album-name, images, and the artist array.
+                return <AlbumInfo 
+                key={album.name}
+                image={album.images[0].url} 
+                name={album.name} 
+                artist={album.artists.map(artist => { // Mapping through the artist array for when having several artists
+                    return (
+                    <a 
+                        key={artist.external_urls.spotify} 
+                        href={artist.external_urls.spotify} 
+                        target="_blank" 
+                        rel="noopener noreferrer" 
+                        className="artists-links">
+                        <span key={artist.name} className="artist-name">{artist.name}</span>
+                    </a>
+                    )
+                })} 
```

Comment:
Consider doing this in another component and naming the prop artists:
`artists={album.artists}`
E.g. in the `<AlbumInfo/>` component, you could mount an `<Artist/>` component:

```
{props.artists.map(item => {
  return (
    <Artist
      key={item.id}
      item={item}
      artistLink={artist.external_urls.spotify} 
      className="artists-links">/>
  )
})}
```
and then have the actual a tag in the `<Artist/>` component